cobalt/browser/h5vcc_metrics/h5vcc_metrics_impl.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COBALT_BROWSER_h5vcc_metrics_H5VCC_METRICS_IMPL_H_
+#define COBALT_BROWSER_h5vcc_metrics_H5VCC_METRICS_IMPL_H_
+
+#include <string>
+
+#include "cobalt/browser/h5vcc_metrics/public/mojom/h5vcc_metrics.mojom.h"
+#include "content/public/browser/document_service.h"
+#include "mojo/public/cpp/bindings/pending_receiver.h"
+
+namespace content {
+  class RenderFrameHost;
```

Comment:
tiny nit: unindent

Reply:
Race condition - I think I fixed this as you were commenting :- )